### ISSUE TITLE
Mitochondrial hypertrophy

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6767]
**Preferred term label:**
Mitochondrial hypertrophy

**Synonyms*

Megamitochondria

**Definition (free text, please give PubMed ID)**
Enlargment of mitochondria. Mitochondrial hypertrophy is not discernible by light microscopy. By electron microscopy (EM), hypertrophic mitochondria have normal cristae and normal matrix density. In contrast, swollen mitochondria display swollen cristae and irregular matrix densities in EM. 
PMID: 32715519
PMID: 29062908  2/2

**Parent term (use hpo.jax.org/app)**
Abnormal mitochondrial shape HP:0012087

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:500013
PMID:32715519   

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
